### PR TITLE
Add support for VMAF

### PR DESCRIPTION
**Pull request by ghost**
Was interested in seeing output from [VMAF](https://github.com/Netflix/vmaf), made it an optional flag because it's a bit slow on my box. Feel free to close request if not useful to you. Thanks!

### CODE REVIEW

File: generate_data.py [hunk 1]
```diff
@@ -268,7 +269,8 @@ def positive_int(num):
 parser.add_argument('--encoded_file_dir', default=None, type=writable_dir)
 parser.add_argument('--frame_offset', default=0, type=positive_int)
 parser.add_argument('--num_frames', default=-1, type=positive_int)
-
+parser.add_argument('--enable-vmaf', dest='vmaf', action='store_true')
```
Comment (pbos) [inline generate_data.py]: Can you sort these in alphabetic order and add default=False instead of parser.set_defaults?

**Comment by pbos** [comment]
Looks great, thanks. Can you also add documentation of these in README.md?

File: generate_data.py (hunk 1 above)
Comment (pbos) [reply generate_data.py]: Alphabetic order is optional, if you want to I can do that in a follow-up too.

**Comment by pbos** [comment]
Please add the documentation entry under Generating Data in README.md, similar to the libyami under Encoders.

**Comment by pbos** [comment]
.. also please add vmaf/ to .gitignore.

**Comment by ghost** [comment]
I decided to refrain from sorting the argparse arguments, because I was unsure if you had a preference regarding the grouping of required and optional ones. Thanks!

**Comment by pbos** [comment]
Thanks! :)

**Pull request merged**